trendmicro_deepsecurity/icon_trendmicro_deepsecurity/actions/deploy_rules/action.py

```diff
@@ -47,23 +47,22 @@ def run(self, params={}):
 
         # Set rules
         response = requests.post(url,
-                                data=json.dumps(data),
-                                headers=post_header,
-                                verify=True)
-        response.close()
+                                data = json.dumps(data),
+                                headers = post_header,
+                                verify = True)
```

Comment:
Same with the keyword argument rule above (apply that to all instances in the project)

Reply:
done